src/store/auth/auth.slice.ts

```diff
@@ -0,0 +1,63 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { AuthState } from '../../types/slices.ts';
+
+const initialState: AuthState = {
+  userId: "",
```

Comment:
accountsContents should be stored under each account it is related to I don't know why we need to separate them